### PR TITLE
Store version1

### PR DESCRIPTION
**Pull request by yermolenko-d**

### CODE REVIEW

File: Store.MVC/App_Start/RouteConfig.cs [hunk 1]
```diff
@@ -13,6 +13,32 @@ public static void RegisterRoutes(RouteCollection routes)
         {
             routes.IgnoreRoute("{resource}.axd/{*pathInfo}");
 
+            routes.MapRoute(null, "",
```
Comment (YaroslavKuchuk) [inline Store.MVC/App_Start/RouteConfig.cs:16]: Why  it is so hard?

File: Store.MVC/Content/bootstrap-reboot.css [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: #212529;
+  text-align: left;
+  background-color: #fff;
+}
+
+[tabindex="-1"]:focus {
+  outline: 0 !important;
```
Comment (YaroslavKuchuk) [inline Store.MVC/Content/bootstrap-reboot.css]: just advice - !important is a bad style of css-ing. But it is not in scope of project.

File: Store.MVC/Controllers/BasketController.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            {
+                Basket = basket,
+                UrlReturn = urlReturn
+            });
+
+        }
+
+        public RedirectToRouteResult AddToBasket(Basket basket, int id, string urlReturn)
+        {
+            Cake cake = repository.Cakes
```
Comment (YaroslavKuchuk) [inline Store.MVC/Controllers/BasketController.cs:61]: Do we have a chance to create a service tier in app?

File: Store.MVC/Controllers/BasketController.cs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            }
+
+            return RedirectToAction("Index", new { urlReturn });
+        }
+
+        public PartialViewResult Summary(Basket basket)
+        {
+            return PartialView(basket);
+        }
+        // GET: Basket
```
Comment (YaroslavKuchuk) [inline Store.MVC/Controllers/BasketController.cs:89]: no comments in code.

If you are using GitHub or something else - what is the purpose to use comments?

File: Store.MVC/Controllers/CakeController.cs [hunk 5]
```diff
@@ -18,18 +20,32 @@ public CakeController(ICakeRepository repository)
...
-                   .OrderBy(c => c.Id)
-                   .Skip((page - 1)*pageSize)
-                   .Take(pageSize));
-        }
+        public ViewResult List(string category, int page = 1)
+        {
+            CakeListView model = new CakeListView
+            {
+                Cakes = cakeRepository.Cakes
+                   .Where(c => category == null || c.Category == category)
```
Comment (YaroslavKuchuk) [inline Store.MVC/Controllers/CakeController.cs:28]: As I mentioned above - create a service including repository as dependency and then move all LINQ methods to it

**Review by YaroslavKuchuk — COMMENTED** [review]
try to fix my comments please

File: Store.MVC/Controllers/BasketController.cs (hunk 4 above)
Comment (yermolenko-d) [reply Store.MVC/Controllers/BasketController.cs:89]: Done. And by the way thanks for explanation.

File: Store.MVC/Controllers/CakeController.cs (hunk 5 above)
Comment (yermolenko-d) [reply Store.MVC/Controllers/CakeController.cs:28]: > As I mentioned above - create a service including repository as dependency and then move all LINQ methods to it

Well, I've been trying to do this. But it takes so much elements in its signature if I will make it as a independed method. So thats a problem...

File: Store.MVC/Content/bootstrap-reboot.css (hunk 2 above)
Comment (yermolenko-d) [reply Store.MVC/Content/bootstrap-reboot.css]: > just advice - !important is a bad style of css-ing. But it is not in scope of project.

I thought about to change all styles to make it looks like my homepage. But I'll do it at the last point i think. Now back-end functional is more important to make.

**Pull request merged**